crates/holochain_p2p/src/test.rs

```diff
@@ -245,7 +245,7 @@ mod tests {
 
     macro_rules! newhash {
         ($p:ident, $c:expr) => {
-            holo_hash::$p::from_raw_36([$c as u8; HOLO_HASH_UNTYPED_LEN].to_vec())
+            holo_hash::$p::from_raw_36([b'3'; HOLO_HASH_UNTYPED_LEN].to_vec())
```

Comment:
I think the tests are expecting to be able to set these hashes to different values... should this be reverted? alternately, this is basically a constant now, should we delete the macro?

Reply:
Yep, good spot. That's `clippy --fix` doing funky things